MQuince/MQuince.Services.Implementation/AppointmentService.cs

```diff
@@ -149,6 +154,21 @@ public IEnumerable<AppointmentDTO> FillFreeInterval(DateTime startTime, DateTime
             return freeAppointments;
         }
 
+        public Appointment CancelAppointment(Guid IdAppointment, DateTime today)
```

Comment:
Method return type sholud be boolean to ensure if Appointment is canceled or not.
You can get current date with DateTime.Now. Method parameter 'today' is not necessary